src/redux/books/book.js

```diff
@@ -1,53 +1,62 @@
-import { v4 as uuid } from 'uuid';
+// import { v4 as uuid } from 'uuid';
+import api from '../../API/api';
 // Actions
 export const ADD_BOOK = 'ADD_BOOK';
 export const REMOVE_BOOK = 'REMOVE_BOOK';
+export const FETCH_BOOKS = 'FETCH_BOOKS';
 
 // state
-const initialState = {
-  books: [
-    {
-      id: uuid(),
-      title: 'The Hunger Games',
-      author: 'Suzanne Collins',
-    },
-    {
-      id: uuid(),
-      title: 'Dune',
-      author: 'Frank Herbert',
-    },
-    {
-      id: uuid(),
-      title: 'Capital in the Twenty-First Century',
-      author: 'S. Piketty',
-    },
+// const initialState = {
+//   books: [
+//     {
+//       id: uuid(),
+//       title: 'The Hunger Games',
+//       author: 'Suzanne Collins',
+//     },
+//     {
+//       id: uuid(),
+//       title: 'Dune',
+//       author: 'Frank Herbert',
+//     },
+//     {
+//       id: uuid(),
+//       title: 'Capital in the Twenty-First Century',
+//       author: 'S. Piketty',
```

Comment:
- [ ] [OPTIONAL] Kindly remove all commented code in your codebase to make it neater and easier to read 🙏